css/homepage.scss

```diff
@@ -92,6 +92,17 @@ $border-type-color: #eff3f6;
       margin: 1em auto;
       font-size: 150%;
   }
+  .ui.cards>.card>.image {
+    padding: 5%;
+    div {
+      overflow: hidden;
+      height: 15rem;
```

Comment:
@MattiSG I took your input but I don't really understand the difference. Could we meet so we explain it ?